index.html

```diff
@@ -28,7 +28,7 @@
 </style>
 
 <h1>Tic Tac Toe</h1>
-
+<h2>Some changes adding a subheading</h2>
```

Comment:
adding a suggestion here
```suggestion
<h2>Some changes adding a subheading</h2>
```